.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,281 @@
+          go test -tags=integ ./tests/integration/pilot/analysis/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ ./tests/integration/pilot/revisions/... -istio.test.skipVM true -p 1 -test.v
+          go test -count=1 -tags=integ ./tests/integration/pilot/cni/...  -istio.test.kube.helm.iopFile $(pwd)/tetrateci/iop-gke-integration.yml  -test.v
+          go test -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  ./tests/integration/telemetry/stats/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/ca_custom_root/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/file_mounted_certs/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/filebased_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtls_first_party_jwt/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtlsk8sca/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_egress/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/webhook/... -istio.test.skipVM true -p 1 -test.v
+
+      - name: gke-cluster-delete
+        run: bash ./tetrateci/cleanup_gke_cluster.sh
+
+  aks-e2e-test:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    name: aks-e2e-test
+    runs-on: ${{ matrix.os }}
+    needs: [create-test-images]
+
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Cloud SDK
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }} 
+
+      - name: set up go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+
+      - name: aks-cluster-create
+        run: bash ./tetrateci/create_aks_cluster.sh
+
+      - name: integration-tests
+        run: |
+          export TAG=$(git describe --tags)
+          go test -tags=integ ./tests/integration/operator/...  -istio.test.skipVM true -p 1  -test.v
```

Comment:
Need to know if there is difference in the tests that are run in different distros? Ideally it should be the same set of tests and it should succeed in all the distros.

Reply:
Yes there are a couple of tests missing in gke and aks compared to eks

Reply:
Need the document and check out why it would not work. Please create an issue.

Reply:
https://github.com/tetratelabs/getistio/issues/75